lightning/src/ln/channelmanager.rs

```diff
@@ -8470,6 +8474,12 @@ macro_rules! create_refund_builder { ($self: ident, $builder: ty) => {
 	}
 } }
 
+/// Defines the maximum number of [`OffersMessage`] to be sent along different reply paths.
```

Comment:
s/reply paths/paths

We are _including_ different reply paths, not sending along them. LDK currently only puts one path in each offer/refund, though we'll send along more than one if given such an offer/refund. And here we are sending the message more than once on each path, each time with a different reply path.